dataset-metadata/metadata.ts

```diff
@@ -19,6 +19,7 @@ export const datasetLabels = {
   gnomad_sv_r4: 'gnomAD SVs v4.0',
   gnomad_cnv_r4: 'gnomAD CNVs v4.0',
   gnomad_r4: 'gnomAD v4.0.0',
+  gnomad_r4_non_ukb: 'gnomAD 4.1 (non-UKB)',
```

Comment:
Very minor -- based on the others in the list, I think we should add a "v" and maybe make it "4.1.0" , as in "`gnomad v4.1.0 (...`" just for consistency with the other entries in the list here

![Screenshot 2024-04-18 at 08 59 12](https://github.com/broadinstitute/gnomad-browser/assets/59549713/2556ade0-c1e0-402e-b4f3-c6f89e9c02c6)